### PR TITLE
Add MFA claims

### PR DESCRIPTION
**Pull request by martimatix**
https://www.notion.so/Create-Pre-Token-Generation-Lambda-Trigger-754d44dd777343458d08ddb0cfdf2cbf

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -9,8 +9,16 @@ provider:
...
+  logRetentionInDays: 14
   stackTags:
     project: datarock-platform
+  iam:
+    role:
+      statements:
+        - Effect: 'Allow'
+          Action:
+            - 'cognito-idp:AdminGetUser'
+          Resource: { "Fn::Sub": "arn:aws:cognito-idp:${AWS::Region}:${AWS::AccountId}:userpool/*" }
```
Comment (carwinz) [inline serverless.yml:21]: could we lock this down to the specific user pool just for "least privilege"? It's innocent enough now, but it'd be easy to start adding other `Action`s...

**Review by carwinz — APPROVED** [review]
Nice and tidy

File: serverless.yml (hunk 1 above)
Comment (martimatix) [reply serverless.yml]: Done.

File: src/cognito.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import AWS from 'aws-sdk';
+
+const cognitoidentityserviceprovider = new AWS.CognitoIdentityServiceProvider();
```
Comment (dil) [inline src/cognito.js:3]: Was it accidental to not camelCase this variable?

**Review by dil — APPROVED** [review]

File: src/cognito.js (hunk 2 above)
Comment (martimatix) [reply src/cognito.js:3]: I copied it from the documentation and didn't realise it wasn't properly cased. 😬

**Pull request merged**